decidim-dev/lib/generators/decidim/dummy_generator.rb

```diff
@@ -37,6 +50,10 @@ def create_dummy_app
         ]
       end
 
+      def set_locales
+        template "locales.rb", "#{dummy_path}/config/initializers/locales.rb"
```

Comment:
Please use `inject_into_file` instead and inject this into `config/application.rb`:

```ruby
config.i18n.available_locales = %w(en ca es)
```

Also, it wouldn't harm to use the occasion to set the default locale:

```ruby
config.i18n.default_locale = :en
```